tests/e2e/scripts/cleanup.sh

```diff
@@ -40,5 +40,28 @@ kill_edgecontroller() {
     fi
 }
 
+cleanup_files(){
+    echo $PWD
+    cd $PWD
+    cd ../../
```

Comment:
Probably we can define the gopath and then find the proper files to delete base on gopath.

Reply:
Thanks.